src/clj/rems/api/applications_v2.clj

```diff
@@ -65,10 +67,20 @@
           :application/form
           :application/licenses))
 
+(mount/defstate all-applications-cache
+  :start (atom {:last-processed-event-id 0
+                :applications nil}))
```

Comment:
core.cache has nice functional caches with various startegies, please use it instead

Reply:
The events are immutable and the injections are not yet cached in state, so for now an atom is enough. I'll keep core.cache in mind when doing further optimizations.